src/Pages/Authentication/Login.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Header } from "../../Components/Header/Header.jsx"
+import { Link } from "react-router-dom";
+import "./Auth.css"
+
+export const Login = () => {
+  return (
+    <>
+    <Header/>
```

Comment:
rather than importing the Header in all pages, you can import in app file above routes

Reply:
Done 37f45c12b0cf181fe5599dfaa2dd4722907bf81e